docs/src/app/(public)/(content)/react/components/tabs/demos/hero/css-modules/index.tsx

```diff
@@ -15,7 +15,7 @@ export default function ExampleScrollArea() {
         <Tabs.Tab className={styles.Tab} value="account">
           Account
         </Tabs.Tab>
-        <Tabs.Indicator className={styles.Indicator} />
+        <Tabs.Indicator className={styles.Indicator} renderBeforeHydration />
```

Comment:
Just double checking: `renderBeforeHydration` isn't required for tabs that don't animate on page load? If yes, this is probably unnecessary in the demos too?

Reply:
> Any reason to not set this problem to true by default? (And even remove the prop, potentially)

Agree here

Reply:
It adds a script to every instance of the Indicator, so the application would grow in size. I can look into including just one instance of the script, though.